src/app/components/HeroSection.jsx

```diff
@@ -16,9 +17,9 @@ export default function HeroSection() {
       <div className="flex font-manrope uppercase flex-col items-start font-semibold text-5xl">
         <div className="flex">
           <span>Let's</span>
-          <button className="px-6 py-3 bg-blue-500 text-white rounded-full ml-3 ">
+          <Button className="px-6 py-3 bg-blue-500 text-white rounded-full ml-3 ">
```

Comment:
some of these classes are already applied to Button component, please clean them up